cmake/BuildLuaTests.cmake

```diff
@@ -64,10 +64,10 @@ ExternalProject_Add(${LUA_TESTS_TARGET_NAME}
         -B <BINARY_DIR>
         -G ${CMAKE_GENERATOR}
         "${LUA_TESTS_CMAKE_FLAGS}"
-  BUILD_COMMAND cd <BINARY_DIR> && ${CMAKE_MAKE_PROGRAM} "${LUA_TESTS}"
+  BUILD_COMMAND cd <BINARY_DIR> && ${CMAKE_MAKE_PROGRAM} "${CAPI_TESTS}"
```

Comment:
Nit: I suppose we may call `cmake --build` instead of `CMAKE_MAKE_PROGRAM`.
Feel free to ignore.

Reply:
`CMAKE_MAKE_PROGRAM` can be `ninja`, `cmake` or anything else. Depends on the configuration produced on configuration stage. See [^1]. So I would not hardcode `cmake` program here.

[^1]: https://cmake.org/cmake/help/latest/variable/CMAKE_MAKE_PROGRAM.html